rules-topics/01_introduction.md

```diff
@@ -62,6 +62,56 @@ Welcome to Chronicles. Whether you've been LARPing for years, or have never seen
 tabard in your life, Chronicles welcomes you. For the newer players, here are a few tips
 to get you started
 
+### Creating a Character
+
+If you are a new player you will start with 100 Skill Points 10 Body and 5 Armor Training.  Skill Points are used to purchase skills like the ability to swing a sword or cast a spell.  You can use the Character Calculator to design your character.  
+* All characters earn 3 Skill Points per day of an event.
+* Characters Under 200 Build gain Skill Points at 2x the rate.
+* All Characters under 200 build can be recreated or rewritten at any time.
+* You can accrue up to 1 extra Skill Point per day by working a NPC shift (See the section on Goblin Points for more details).
+
+If you have a character in another Larp System please see our Transfer Policy if you want to transfer your character to Chronicles.
```

Comment:
Can we update "Larp" to "LARP" here and lowercase "System"?  Could also use a comma following "system."

Reply:
Updated